src/components/Api.js

```diff
@@ -0,0 +1,174 @@
+export default class Api {
+  constructor(options) {
+    //constructor
+    this.baseUrl = options.baseUrl;
+    this._headers = options.headers;
+    //this._authorization = this.headers["authorization"];
+  }
+
+  getInitialCards(id) {
+    return fetch(`${this.baseUrl}/${id}`, {
+      headers: this._headers,
+    })
+      .then((res) => {
```

Comment:
The code for processing server response gets duplicated. You should declare a method and reuse it

```js
_processResponse = (res) => { 
      if (res.ok) { 
        return res.json(); 
      } 
      return Promise.reject(`Error: ${res.status}`); 
}

getInitialCards() { 
  ...
  .then(this._processResponse)
}
```